From 3a00a6e87ee431a4feafb511bd4dd3bae1418cef Mon Sep 17 00:00:00 2001
From: Alexandru Costache <alexandru@balena.io>
Date: Fri, 29 May 2020 16:54:48 +0200
Subject: [PATCH] kconfig: Add fsuuid command

If config does not exist in Kconfig,
build will either fail, or not enable the config

Upstream-status: backport
Signed-off-by: Alexandru Costache <alexandru@balena.io>
---
 Kconfig | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/Kconfig b/Kconfig
index 02f954d64e..0240d72faa 100644
--- a/Kconfig
+++ b/Kconfig
@@ -325,6 +325,11 @@ config ARCH_FIXUP_FDT_MEMORY
 	  used for booting OS with different memory setup where the part of
 	  the memory location should be used for different purpose.
 
+config CMD_FS_UUID
+        bool "fsuuid command"
+        help
+          Enables fsuuid command for filesystem UUID.
+
 endmenu		# Boot images
 
 source "api/Kconfig"
-- 
2.17.1

